nx-X11/config/cf/xf86site.def

```diff
@@ -1,479 +0,0 @@
-XCOMM $XFree86: xc/config/cf/xf86site.def,v 3.186 2003/06/25 18:06:22 eich Exp $
-/******************************************************************************/
-/*
- * This file is to provide a quick method for most people to change the
- * behaviour of their XFree86 installation without having to fully
- * understand the workings of site.def and all the various '.cf' files.
- *
- * In the vast majority of cases, it should not be necessary to use this
- * file at all or change it in any way.
- *
- * This file is divided into two sections.  The first contains settings
- * that end-users might reasonably change.  The second contains settings
- * that developers might want to change.
- *
- * IMPORTANT NOTE:  In all cases changing the defaults may cause problems
- * and/or unexpected side-effects.  If you don't understand what a setting
- * does, then it's best to not change it.  If you make changes and have
- * problems, verify that those problems are also present when using an
- * empty host.def file and unchanged version of this file before reporting
- * them.
- *
- * A good way to use this file is to copy it to host.def, and make the
- * changes there.  That way, future patches to this file won't fail.
- * The host.def file will never be patched.
- *
- * The distributed version of this file MUST contain no uncommented
- * definitions.  Default definitions belong in xfree86.cf, or <vendor>.cf
- * files.
- */
-/******************************************************************************/
-
-/*
- * If you have build-specific modifications in your host.def file, but
- * want an empty host.def file installed when doing 'make install',
- * uncomment the following
- *
-#define InstallEmptyHostDef
- */
-
-/*
- * Which servers to build.  There is now only one XFree86 server.
- * It can be disabled by the following.
- *
-#define XF86Server		NO
```

Comment:
Can also be removed in
  -  `nx-X11/programs/Xserver/Xext/Imakefile`
  - `nx-X11/programs/Xserver/xkb/Imakefile`